### PR TITLE
docs: add windows support communication to readme

### PR DESCRIPTION
**Pull request by pswu11**
Added some communication in README for windows support as suggested in [https://github.com/jina-ai/examples/issues/246](url)

### CODE REVIEW

**Review by hanxiao — CHANGES_REQUESTED** [review]
This is written in docs already, https://docs.jina.ai/chapters/install/via-pip.html?highlight=windows#on-windows-and-other-oses

**Review by bwanglzu — DISMISSED** [review]
do you think it's better to rephrase Windows environment to Microsoft Windows?

**Comment by alexcg1** [comment]
> This is written in docs already, https://docs.jina.ai/chapters/install/via-pip.html?highlight=windows#on-windows-and-other-oses (@hanxiao)

Yup, it's already in the docs. I think it'd be good to reference that page at top of each example so users don't waste their time and get a bad impression of us

**Comment by alexcg1** [comment]
I think:

- Let's add it in the core repo, but not so obviously. Perhaps under 'Get Started'
- Add to top of each example in a more obvious way

In both cases, link to the page @hanxiao mentioned

File: README.md [hunk 1]
```diff
@@ -92,6 +92,10 @@ We provide a universal Docker image that supports multiple architectures (includ
 docker run jinaai/jina --help
 ```
 
+### Windows Support
```
Comment (bhavsarpratik) [inline README.md:95]: Can use this emoji 
https://emojipedia.org/warning/

**Comment by pswu11** [comment]
@hanxiao @alexcg1 
I revised the communication accordingly. Instead of having an independent section, just a short sentence that links directly to the section.

**Comment by alexcg1** [comment]
Let's change the text:

Old:

```
If you are Windows user, see [this section](https://docs.jina.ai/chapters/install/via-pip.html?highlight=windows#on-windows-and-other-oses)
```

New: 

```
⚠️ We don't currently support Windows, but we're working on it! [See here for more information](https://docs.jina.ai/chapters/install/via-pip.html?highlight=windows#on-windows-and-other-oses).
```

That lets them know immediately we don't support Windows, instead of them having to click the link to find out

**Comment by pswu11** [comment]
> Let's change the text:
> 
> Old:
> 
> ```
> If you are Windows user, see [this section](https://docs.jina.ai/chapters/install/via-pip.html?highlight=windows#on-windows-and-other-oses)
> ```
> 
> New:
> 
> ```
> ⚠️ We don't currently support Windows, but we're working on it! [See here for more information](https://docs.jina.ai/chapters/install/via-pip.html?highlight=windows#on-windows-and-other-oses).
> ```
> 
> That lets them know immediately we don't support Windows, instead of them having to click the link to find out

@alexcg1 adjusted based on our discussion

**Comment by alexcg1** [comment]
@bwanglzu I don't think we need to specify **Microsoft** Windows. It's not there's any other Windows out there (except X Windows, but that's a whole other thing)

**Comment by alexcg1** [comment]
@pswu11 Re your commit message, you'll need to rewrite to make it lower-case otherwise it may fail linting. [Instructions here](https://docs.github.com/en/free-pro-team@latest/github/committing-changes-to-your-project/changing-a-commit-message). Let me know if you need help!

**Comment by alexcg1** [comment]
LGTM. @hanxiao if all good with you we can merge (normally I wouldn't ask about something so small, but this is on our main README after all)

**Review by hanxiao — APPROVED** [review]

**Pull request merged**